docs/using/experimental.md

```diff
@@ -0,0 +1,69 @@
+# Experimental features
+
+Some Interchange features are available in the public API but NOT suitable for production. These are the sharper edges of the project that we hope to make more stable and reliable in the future. Generally these features are somewhat complete and partially tested, but not completely finished and in need of more extensive testing before their use is considered safe.
+
+In order to facilitate testing these features, though, they are included in the package on an "opt-in" basis. These features are accessible by setting the environment variable `INTERCHANGE_EXPERIMENTAL=1`. By setting this variable the user accepts that they are accessing a portion of the codebase that is **not stable**, highly likely to produce **inaccurate results**, and not guaranteed to run without error. For these reasons, all features tagged as experimental are NOT recommended for production.
+
+## Current set of experimental features
+
+See each function's docstring for more detailed information.
+
+* `Interchange.from_gromacs`: Import data from GROMACS files.
+* `Interchange.from_openmm`: Import data from OpenMM objects.
+* `Interchange.__add__`: Combine two `Interchange` objects (this is called under the hood when using the `+` operator with two `Interchange` objects).
+
+## Using experimental features
+
+Testing and user feedback on experimental features is very welcome! To access an experimental feature, set  `INTERCHANGE_EXPERIMENTAL=1` when starting a Python interpreter:
+
+```shell
+$ INTERCHANGE_EXPERIMENTAL=1 python my_script.py
+...
+```
+
+or Jupyter notebook or equivalent
+
+```shell
+$ INTERCHANGE_EXPERIMENTAL=1 jupyter-notbook my_notebook.ipynb
+...
+```
+
+Feedback of any sort is useful - even (and especially) if something crashes early or produces surprising results. Please raise an [issue on GitHub](https://github.com/openforcefield/openff-interchange/issues/new) or email <info@openforcefield.org> sharing what you are trying to do and what happened instead.
```

Comment:
Feel free to suggest a different email (`support@` ?) or none at all. Receiving support emails is not my favorite means of communication but I understand some people prefer it and/or might not be able to post publicly.

Reply:
I wouldn't list an email at all.